backend/keepalived/keepalived_test.go

```diff
@@ -246,8 +246,6 @@ func TestCreateKeepaliveEvent(t *testing.T) {
 	assert.Equal(t, "keepalive", keepaliveEvent.Check.Name)
 	assert.Equal(t, uint32(60), keepaliveEvent.Check.Interval)
 	assert.Equal(t, []string{"keepalive"}, keepaliveEvent.Check.Handlers)
-	assert.Equal(t, uint32(1), keepaliveEvent.Check.Status)
+	assert.Equal(t, uint32(0), keepaliveEvent.Check.Status)
```

Comment:
It _appears_ to be fine here, but I'm a bit uncomfortable with changing the semantics of `createKeepaliveEvent()` from "the created event's check status is 1 (failing)" to "the created event's check status is 0 (passing)".

Reply:
Yeah, that's why I said it appears to be fine. Generally speaking though, those kind of changes always trigger paranoid mode in my head :)

Agreed with the naming convention! No need for a change now though: it's clear we're on the same page and that function is not exported or used in a lot of places anyway.